### DISCUSSION TITLE
How to deploy a custom favicon

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1578] [Help]
Hi i am wondering if there is a way i can change the favicon on my app from the chrome default of the world to my company logo. 

Thank you for the help!
![image](https://user-images.githubusercontent.com/80058238/119851343-2dba0d00-bec3-11eb-922b-20467f09a12f.png)

**Comment by mjashanks** [comment]
Unfortunately, this is not possible right now. You can change the logo that appears in the top left of the apps list screen - via `Organisation`. 

We should add the option for a favicon in here. I think it makes sense to be global, not per app?

**Reply by KernOil** [reply]
yes I believe global would work perfectly. 
Thank you for the update.

**Reply by NicolasFabre81** [reply]
Hi @mjashanks , I think a per-app favicon would make sense because one can be a user of several BB apps...

**Comment by wildsonc** [comment]
If you use a nginx proxy you can serve the .ico, example:

location = /favicon.ico {
        alias /var/www/media/images/favicon.ico;
}

![image](https://user-images.githubusercontent.com/72408418/138765591-9b985ba4-b507-4206-b092-2a00a36b399b.png)

**Reply by Kuuchuu** [reply]
@wildsonc I added this into my NGINX config. The console error about the missing favicon.ico is gone, but the favicon still does not show. Inspecting the html shows the favicon href is either empty or set to "unknown", which seems to be taking precedence over the default /favicon.ico path? Am I missing something?

**Reply by vicam001** [reply]
Hi @wildsonc , may you please provide more information on how to change the favicon? I am a newby in budibase and using the out of the box docker-compose self hosting, I would appreciate 1-2 tips to get started.

Thank you very much!
